docker/debug-pod/Dockerfile

```
@@ -56,6 +56,13 @@ RUN ARCH=$(dpkg --print-architecture) && \
     ./google-cloud-sdk/install.sh && \
     rm -rf google-cloud-cli-linux-*
 
+# s5cmd cli Install
+RUN wget https://github.com/peak/s5cmd/releases/download/v2.3.0/s5cmd_2.3.0_Linux-64bit.tar.gz && \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)